includes/Modules/AdSense.php

```diff
@@ -347,32 +347,34 @@ protected function amp_content_add_auto_ads( $content ) {
 	}
 
 	/**
-	 * Returns the mapping between available datapoints and their services.
+	 * Gets map of datapoint to definition data for each.
 	 *
-	 * @since 1.0.0
+	 * @since n.e.x.t
 	 *
-	 * @return array Associative array of $datapoint => $service_identifier pairs.
+	 * @return array Map of datapoints to their definitions.
 	 */
-	protected function get_datapoint_services() {
+	protected function get_datapoint_definitions() {
 		return array(
-			// GET / POST.
-			'connection'     => '',
-			'account-id'     => '',
-			'client-id'      => '',
-			'use-snippet'    => '',
-			'account-status' => '',
-			// GET.
-			'account-url'    => '',
-			'reports-url'    => '',
-			'notifications'  => '',
-			'tag-permission' => '',
-			'accounts'       => 'adsense',
-			'alerts'         => 'adsense',
-			'clients'        => 'adsense',
-			'urlchannels'    => 'adsense',
-			'earnings'       => 'adsense',
-			// POST.
-			'setup-complete' => '',
+			'GET:accounts'        => array( 'service' => 'adsense' ),
```

Comment:
`GET:accounts` should be after `GET:account-url` and before `GET:alerts`.

Reply:
Updated.